supporting/emissions.py

```diff
@@ -434,6 +434,16 @@ def check_geo_types(self):
             
             return 
     
+    def get_max_emitting_area():
```

Comment:
I think the issue here is you need add `self` inside the function definition (put simpler: change line 437 to `def get_max_emitting_area(self):`)